framework/packages/abstract-client/src/account.rs

```diff
@@ -327,6 +335,22 @@ impl<Chain: CwEnv> Account<Chain> {
         }
     }
 
+    /// Upgrades the account to the new version
+    /// MIgrates manager and proxy modules to their respective new versions
+    pub fn upgrade(&self) -> AbstractClientResult<()> {
+        self.abstr_account.manager.upgrade(vec![
+            (
+                ModuleInfo::from_id("abstract:manager", ModuleVersion::Latest)?,
+                Some(to_json_binary(&abstract_core::manager::MigrateMsg {})?),
+            ),
+            (
+                ModuleInfo::from_id("abstract:proxy", ModuleVersion::Latest)?,
+                Some(to_json_binary(&abstract_core::proxy::MigrateMsg {})?),
+            ),
+        ])?;
```

Comment:
Let's reuse names from core registry

```suggestion
            (
                ModuleInfo::from_id(abstract_core::registry::MANAGER, ModuleVersion::Latest)?,
                Some(to_json_binary(&abstract_core::manager::MigrateMsg {})?),
            ),
            (
                ModuleInfo::from_id(abstract_core::registry::PROXY, ModuleVersion::Latest)?,
                Some(to_json_binary(&abstract_core::proxy::MigrateMsg {})?),
            ),
        ])?;
```